app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated.")
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date < datetime.date.today():

```

Comment:
Ensure that `expiration_date` is a valid `date` object before comparing it with `datetime.date.today()`. Consider adding a check or conversion if necessary to prevent potential errors.